### PR TITLE
[139]: Replace buttons

### PR DESCRIPTION
**Pull request by six7**
Replaces all occurences of legacy buttons with shared ones

### CODE REVIEW

**Review by LukeFinch — APPROVED** [review]

File: packages/tokens-studio-for-figma/src/stories/ThemeStyleManagementCategory.stories.mdx [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import { Meta, Story, ArgsTable, Canvas, Preview } from '@storybook/addon-docs';
 import { ThemeStyleManagementCategory } from '@/app/components/ManageThemesModal/ThemeStyleManagementCategory';
-import ColorDisplayBubble from '@/app/components/ColorDisplayBubble';
```
Comment (SorsOps) [inline packages/tokens-studio-for-figma/src/stories/ThemeStyleManagementCategory.stories.mdx:3]: Why was this removed?

Comment (six7) [reply packages/tokens-studio-for-figma/src/stories/ThemeStyleManagementCategory.stories.mdx:3]: It wasn't in use anymore. We removed that a while ago but seems like we never removed the code.
